app/src/main/java/com/example/tasky/agenda/domain/util/Constants.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.tasky.agenda.domain.util
+
+const val MAX_SIZE_BYTES = 1_000_000
+const val MAX_NUMBER_OF_PHOTOS = 10
+const val COMPRESS_QUALITY = 50
+const val OPEN_FILE_MODE = "r"
```

Comment:
This looks very much like an implementation detail - reading a file (and defining how that works) is none of domain's business 😄